README.org

```diff
@@ -69,6 +65,22 @@ There is an option to have your user credentials (email address and password) sa
 The default is not to do this because if not properly configured it would save these unencrypted which is not a good default to have.
 Customize the variable =mastodon-auth-source-file= if you want to enable this feature.
 
+*** 2 Factor Auth
+**** Overview
+After setting up =mastodon-instance-url= set the following in your
+=.emacs= or =customize=:
+
+1. =mastodon-auth-mechanism=
+
+**** Set up
+#+BEGIN_SRC emacs-lisp
+  (setq mastodon-auth-mechanism "oauth2")
+#+END_SRC
+
+The OAuth2 library uses =plstore= to securely store the credentials.
```

Comment:
Is that plstore location configurable?  Maybe mention the appropriate var?

Reply:
Added info on steps.

Reply:
Seems the var is `oauth2-token-file`, we could/should mention that explicitly here.

Actually, could we just temporarily around oauth operations let `oauth2-token-file` to be the same value as `mastodon-auth-source-file` and thereby use only one secrets storage?

Reply:
I had used `defvaralias` for a while. But I decided to go against it. Since this is a generic library which can be used as a dependency by other packages, I think aliasing it will not be a good idea.

Is there any alternative approach so that the global env is not spoiled?